eZ/Bundle/EzPublishCoreBundle/ApiLoader/RepositoryFactory.php

```diff
@@ -98,6 +100,7 @@ public function buildRepository(PersistenceHandler $persistenceHandler, SearchHa
                     'policyMap' => $this->policyMap,
                 ),
                 'languages' => $this->configResolver->getParameter('languages'),
+                'content' => ['default_version_archive_limit' => $config['options']['default_version_archive_limit']],
```

Comment:
called it `default` as intention is to make room for additional per content type version limit again in the future, and in that context this will serve as the default if nothing is specified for a content type.